### PR TITLE
MSVC Fixes

### PR DESCRIPTION
**Pull request by PTDreamer**
Fixes compile and segfault

### CODE REVIEW

File: ground/uavobjgenerator/main.cpp [hunk 1]
```diff
@@ -29,7 +29,7 @@
 #include <QStringList>
 #include <iostream>
 
-#include <unistd.h>	/* For symlink(2) */
+//#include <unistd.h>	/* For symlink(2) */
```
Comment (kubark42) [inline ground/uavobjgenerator/main.cpp]: This should be removed.

Comment (PTDreamer) [inline ground/uavobjgenerator/main.cpp]: @mlyle added it for reasons I can't explain, I'll remove it after he reviews this.

**Comment by tracernz** [comment]
Thanks for doing this @PTDreamer. :-) Getting a Jenkins slave building with MSVC will avoid this happening again, and give us faster Windows builds.

File: ground/uavobjgenerator/main.cpp (hunk 1 above)
Comment (kubark42) [inline ground/uavobjgenerator/main.cpp]: Ah.

**Comment by kubark42** [comment]
LGTM, ready to merge as soon as @mlyle and @PTDreamer decide what to do about the ```unistd.h``` include.

File: ground/uavobjgenerator/main.cpp (hunk 1 above)
Comment (mlyle) [inline ground/uavobjgenerator/main.cpp]: It can be removed.  In an earlier version of the fixes I used symlink(2) and I guess this crept into the PR.

**Comment by tracernz** [comment]
LGTM

**Pull request merged**